internal/cmd/clone/clone.go

```diff
@@ -3,12 +3,36 @@
 package clone
 
 import (
+	"strings"
+
+	"github.com/gittuf/gittuf/internal/cmd/common"
 	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/gittuf/gittuf/internal/tuf"
 	"github.com/spf13/cobra"
 )
 
 type options struct {
-	branch string
+	branch           string
+	expectedRootKeys PathList
+}
+
+// PathList is a custom type to represent a list of paths
+type PathList []string
```

Comment:
this applies to a bunch of commands, I think? Can we move this to cmd/common? Note: we can update the other commands to use this in a separate PR to keep PRs clean, but let's put it in cmd/common from the start.

Reply:
Also, what does this mean for fulcio / gpg keys which may not be paths?

Reply:
I added a larger description in the expected root-key's description that gpg and fulcio keys should be passed in as the keys themselves.